distributed/comm/tests/test_ucx.py

```diff
@@ -156,6 +154,10 @@ def test_ucx_specific():
 
 @gen_test()
 def test_ucx_deserialize():
+    try:
+        from .test_comms import check_deserialize
+    except Exception:
```

Comment:
Minor comment: could you catch the specific `OSError` exception here and add a small comment about where it's coming from

Reply:
We could do that. Unfortunately I don't know if that's the only error we see here. At the end of the day, this is just a workaround. We really want to fix the underlying issue so this test runs. Do you know why we would be seeing this error in the first place?

Reply:
After thinking about it, I decided to drop the `skip` altogether and just added a comment. Now that the import is local to this test, only this test will error (as opposed to breaking the whole test suite, current behavior). Perhaps seeing this error in this one test will motivate us to either fix the test or fix the boxes that have issues. Whichever solution we find is appropriate.

Reply:
Sorry, after spending a few minutes looking through the code run when importing `check_deserialize` I wasn't able to figure out why the `OSError` is getting raised

Reply:
No worries. I'm equally mystified 🙂